.github/actions/install-oppia-dependencies/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Install Dependencies'
+description: 'Installs wheel and third-party python libs.'
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup python by installing wheel
+      run: pip install wheel==0.35.0
+      shell: bash
```

Comment:
Optional: can we write shell before the run? (Makes it more readable i.e, first the info about the shell and then what to run.)

Reply:
The canonical way to write GitHub Actions workflows is to put the shell at the end as shown in the [documentation](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-running-a-script-using-bash). I think it's best to stick with this standard